stories/Navigation.stories.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import Navigation from "../src/components/navigation/Navigation";
+import { storiesOf } from "@storybook/react";
+
+class NavigationTest extends Component {
+    render() {
+        return <Navigation />;
+    }
+}
+
+storiesOf("Navigation", module).add("with navigation", () => (
```

Comment:
Could save it for another PR, but there's actually a new format that Storybook is recommending on newer versions.   https://storybook.js.org/docs/formats/component-story-format/